tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -192,16 +205,88 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
+                                  string(name: 'GOTEST_TESTCASE', value: "${k3sEC2TestCase}") ]
+
+                      goK3SAZParams = [ string(name: 'CATTLE_TEST_URL', value: "${CATTLE_TEST_URL_2}"),
+                                  string(name: 'ADMIN_TOKEN', value: "${ADMIN_TOKEN_2}"),
+                                  string(name: 'USER_TOKEN', value: "${USER_TOKEN_2}"),
+                                  string(name: 'RANCHER_SERVER_VERSION', value: "${rancher_version}"),
+                                  string(name: 'TIMEOUT', value: "${TIMEOUT}"),
+                                  text(name: 'CONFIG', value: "${provision-config.yaml}"),
+                                  string(name: 'REPO', value: "https://github.com/rancher/rancher.git"),
+                                  string(name: 'BRANCH', value: branch),
+                                  string(name: 'TEST_PACKAGE', value: "provisioning/k3s"),
+                                  string(name: 'GOTEST_TESTCASE', value: "${k3sAZTestCase}") ]                  
+
+                      goK3SCustomParams = [ string(name: 'CATTLE_TEST_URL', value: "${CATTLE_TEST_URL_2}"),
+                                  string(name: 'ADMIN_TOKEN', value: "${ADMIN_TOKEN_2}"),
+                                  string(name: 'USER_TOKEN', value: "${USER_TOKEN_2}"),
+                                  string(name: 'RANCHER_SERVER_VERSION', value: "${rancher_version}"),
+                                  string(name: 'TIMEOUT', value: "${TIMEOUT}"),
+                                  text(name: 'CONFIG', value: "${provision-config.yaml}"),
+                                  string(name: 'REPO', value: "https://github.com/rancher/rancher.git"),
+                                  string(name: 'BRANCH', value: branch),
+                                  string(name: 'TEST_PACKAGE', value: "provisioning/k3s"),
+                                  string(name: 'GOTEST_TESTCASE', value: "${k3sCustomTestCase}") ]
+
+                      // Rancher server 1 is used to test RKE1, RKE2 and K3s clusters.
                       // DO Note: https://github.com/rancher/qa-tasks/issues/318
                       // jobs["do"] = { build job: 'rancher-v3_ontag_do_certification', parameters: params }
                       jobs["ec2"] = { build job: 'rancher-v3_ontag_ec2_certification', parameters: params }
                       jobs["az"] = { build job: 'rancher-v3_ontag_az_certification', parameters: params }
                       jobs["custom"] = { build job: 'rancher-v3_ontag_custom_certification', parameters: params}
+                      jobs["go-rke2-ec2"] = { build job: 'go-automation-freeform-job', parameters: goRKE2EC2Params }
+                      jobs["go-rke2-az"] = { build job: 'go-automation-freeform-job', parameters: goRKE2AZParams }
+                      jobs["go-rke2-custom"] = { build job: 'go-automation-freeform-job', parameters: goRKE2CustomParams }
```

Comment:
Also these jobs are just provisioning the clusters correct? When are the actual tests being ran against them?

Reply:
The certification test provision the cluster then run certain tests against the clusters.